testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/jpa/persistence.xml

```diff
@@ -11,7 +11,7 @@
             <property name="hibernate.hbm2ddl.auto" value="update"/>
             <property name="hibernate.cache.use_second_level_cache" value="true"/>
             <property name="hibernate.cache.use_query_cache" value="true"/>
-            <property name="hibernate.cache.infinispan.entity.cfg" value="entity-replicated"/>
+            <!--property name="hibernate.cache.infinispan.entity.cfg" value="entity-replicated"/-->
```

Comment:
Consider deleting this altogether instead of commenting out.

Reply:
Indeed.